modules/hardware/x86_64-linux.nix

```diff
@@ -25,6 +25,16 @@ in
       hardware.enableRedistributableFirmware = true;
       hardware.enableAllFirmware = true;
 
+      # Enable pulseaudio systemwide support for host
```

Comment:
Maybe move these into targets/lenovo-x1-carbon.nix for now? (As a temporary solution)